### PR TITLE
Update ezil.json

### PR DESCRIPTION
**Pull request by R2D2me**
Single coin pool urls added

### CODE REVIEW

File: pool_templates/ezil.json [hunk 1]
```diff
@@ -64,7 +64,52 @@
               "cn.ezil.me:25443",
               "cn.ezil.me:35443"
             ]
-          }
+          },
+          {
+            "geo": "Europe-ETH only",
```
Comment (Snuff) [inline pool_templates/ezil.json:69]: Could you add spaces around `—`? @R2D2me

Comment (R2D2me) [reply pool_templates/ezil.json:69]: @Snuff "Europe - ETH only" ?

Comment (Snuff) [reply pool_templates/ezil.json:69]: Aha @R2D2me

**Comment by Snuff** [comment]
LGTM 👍

**Pull request merged**